src/styles/card.css

```diff
@@ -0,0 +1,113 @@
+body {
+  font-family: Roboto, sans-serif;
+  --large-text-color: #060b35;
+  --small-text-color: #616070;
+  --button-color: #00acdc;
+}
+
+h2,
+p {
+  margin: 0;
+  padding: 0;
+  display: inherit;
+  font-size: inherit;
+  margin-block-start: inherit;
+  margin-block-end: inherit;
+  margin-inline-start: inherit;
+  margin-inline-end: inherit;
+  font-weight: inherit;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid white;
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__img img {
+  height: 134px;
+  width: 160px;
+  padding-top: 32px;
+}
+
+.card__description {
+  padding: 40px 16px 16px;
```

Comment:
You can add padding to whole card 32px 16px 16px and add margins between elements because it's external indents